### ISSUE TITLE
User Feedback System within GrepThink

### ISSUE THREAD

**Issue by rmonroe** [grepthink/grepthink#202]
- Allow users to send error reports with stack trace
- Inlcude some kind of link to our emails in order to get a faster repsonse on our end
- Allow for feture suggestions

**Comment by Kapn** [comment]
Could probably be its own repository.

**Comment by rmonroe** [comment]
I agree, what if rather then build it into the site what if we build it as a separate module and import or link it to the main site. Since it's pretty self-contained and could have some other features. Then we could also know that we can build stuff this way
